tasks/gulp/backstop-config.js

```diff
@@ -78,7 +78,7 @@ module.exports = ({ host, port, components }) => ({
     ci_report: 'backstop_data/ci_report',
   },
   report: ['browser'],
-  engine: 'puppeteer',
+  engine: 'playwright',
```

Comment:
the apis we are using are compatible, no idea if there are some we have coded but aren't using that won't be